bucketeer/src/main/kotlin/io/bucketeer/sdk/android/BKTEvaluationDetail.kt

```diff
@@ -0,0 +1,85 @@
+package io.bucketeer.sdk.android
+
+import io.bucketeer.sdk.android.internal.util.contains
+import org.json.JSONObject
+
+data class BKTEvaluationDetail<T>(
+  var id: String,
```

Comment:
I just found out that we won't need this field to support the OpenFeature SDK.

```suggestion
```

Reply:
updated in https://github.com/bucketeer-io/android-client-sdk/pull/180/commits/bbb4bcede0e56f8c44a568d11cbc50c9bd57472b